test/passing/infix_bind.ml

```diff
@@ -23,36 +23,27 @@ f x
 ;;
 f x >>= fun y ->
 g y >>= fun () ->
-f x >>= fun y ->
-g y >>= fun () ->
-f x >>= fun y ->
-g y >>= fun () ->
-y ()
+f x >>= fun y -> g y >>= fun () -> f x >>= fun y -> g y >>= fun () -> y ()
```

Comment:
This formatting is weird. They should all break.

Reply:
I think if the chain is long, there is a good chance one of the bodies will be long enough to break the line.
So the cases we want to improve are indeed the smaller ones.

Reply:
So do you think we should not merge this PR?

Reply:
I find this case a bit problematic. What make it hard to fix ?

Reply:
It should be better now.